bin/reth/src/lib.rs

```diff
@@ -43,3 +44,38 @@ struct NetworkOpts {
     #[arg(long, value_delimiter = ',')]
     bootnodes: Option<Vec<NodeRecord>>,
 }
+
+/// Parameters for configuring the network more granularly via CLI
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Rpc")]
+struct RpcServerOpts {
+    #[arg(long, default_value = "true")]
+    http: bool,
+
+    #[arg(long = "http.addr")]
+    http_addr: Option<String>,
+
+    #[arg(long = "http.port")]
+    http_port: Option<u128>,
+
+    #[arg(long = "http.api", value_delimiter = ',')]
+    http_api: Option<Vec<RethRpcModule>>,
+
+    #[arg(long, default_value = "true")]
+    ws: bool,
```

Comment:
same as `http`

Reply:
``` 
--http                              Enable the HTTP-RPC server
```
That's how Geth documented the arg, so I'm guessing passing just --http is default true.

They also have http/ws addr and port defaults set as "localhost" and "8545", don't know if I should set those defaults too.

Reply:
it's `false` otherwise `--http` would be meaningless, so this should also default to false for the time being.